### PR TITLE
Add data-automation-id to woocommerce events [MQ-163]

### PR DESCRIPTION
**Pull request by amine-mp**
[[MQ-163]](https://mailpoet.atlassian.net/browse/MQ-163)

### CODE REVIEW

File: assets/js/src/newsletters/types/automatic_emails/event.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ class AutomaticEmailEvent extends React.PureComponent {
           className="button button-primary"
           disabled={disabled}
           onClick={!disabled ? _.partial(this.props.eventsConfigurator, event.slug) : null}
+          data-automation-id={event.slug}
```
Comment (costasovo) [inline assets/js/src/newsletters/types/automatic_emails/event.jsx]: Maybe we could add some prefix here e.g. `setup-button-` What do you think?

Comment (amine-mp) [reply assets/js/src/newsletters/types/automatic_emails/event.jsx]: I think prefixing it with `create_` would be more consistent with what we have on newsletter types page (`create_standard`, `create_woocommerce`, ...).

Comment (pavel-mailpoet) [reply assets/js/src/newsletters/types/automatic_emails/event.jsx]: I agree some prefix would be good. Those ids should be unique so the longer the better.

**Comment by pavel-mailpoet** [comment]
The build failed 3 times in a row on this branch. Could you please take a look at it before I move this PR to QA?

**Pull request merged**